tests/models/unets/test_models_unet_motion.py

```diff
@@ -65,11 +65,12 @@ def output_shape(self):
 
     def prepare_init_args_and_inputs_for_common(self):
         init_dict = {
-            "block_out_channels": (32, 64),
+            "block_out_channels": (16, 32),
```

Comment:
Possible to make smaller?

Reply:
There's a hard-coded 32 value somewhere in the `TransformerSpatioTemporalModel` module. It will lead to:

```bash
FAILED tests/models/unets/test_models_unet_motion.py::UNetMotionModelTests::test_output - ValueError: num_channels must be divisible by num_groups
```

So, a little infeasible at the moment.  But I have made a best effort to reduce the other dimensions (such as sample size, number of frames, cross attention dim, etc.). Hopefully that cuts it?